tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Is this a formatter config file or a compiler config file only?

Reply:
this specifies the typescript compiler to be used when we will use hardhat to deploy/upgrade the contract